packages/contentstack-config/src/commands/config/get/rate-limit.ts

```diff
@@ -0,0 +1,52 @@
+import { cliux, configHandler, isAuthenticated } from '@contentstack/cli-utilities';
+import { Command } from '@contentstack/cli-command';
+import { handleErrorMsg } from '../../../utils/interactive';
+import { RateLimitConfig } from '../../../interfaces';
+
+export default class RateLimitGetCommand extends Command {
+
+  static examples = ['$ csdx config:get:rate-limit'];
+
+  async run() {
+    try {
+      if (!isAuthenticated()) {
+        const err = { errorMessage: 'You are not logged in. Please login with command $ csdx auth:login' };
+        handleErrorMsg(err);
+      }
+      const rateLimits = configHandler.get('rateLimits') || {};
+      const rateLimitData: RateLimitConfig = rateLimits.rateLimit || {};
```

Comment:
@harshithad0703 , pls check the structure properly. code should be like this

    const rateLimit = configHandler.get('rateLimit') || {};
    const formatLimit = (limit) => limit ? `${limit.value}(${limit.utilize}%)` : '0';

    const tableData = Object.entries(rateLimitData).map(([org, limits]: [string, RateLimitConfig]) => ({
    Org: org === 'default' ? 'default' : org,
    'Get Limit': formatLimit(limits.getLimit),
    Limit: formatLimit(limits.limit),
    'Bulk Limit': formatLimit(limits.bulkLimit),
}));

Reply:
updated the structure